python/llm/src/bigdl/llm/utils/utils.py

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import sys
+from bigdl.llm.utils.common import invalidInputError, invalidOperationError
+
+
+def get_avx_flags():
+    avx = ""
+    if sys.platform != "win32":
+        import subprocess
+        msg = subprocess.check_output(["lscpu"]).decode("utf-8")
+        if "avx512_vnni" in msg:
```

Comment:
we can support avx512 without avx512_vnni since this PR: <https://github.com/analytics-zoo/bloomz.cpp/commit/affa28b2cbfecc7d8d6feeb9b3a3fe35b1c7c070>

Reply:
Any plans for llama and neox?

Reply:
llama and neox already support it too

Reply:
Updated in https://github.com/intel-analytics/BigDL/pull/8266